backend/no_waste.py

```diff
@@ -26,24 +26,48 @@ class Recipe(db.Model):
     recipe_name = db.Column(db.String(255))
     recipe = db.Column(db.Text)
 
+    @property
+    def serialize(self):
+        return {
+            "title": self.recipe_name,
+            "imgUrl": None,
+            "description": self.recipe,
+            "ingredients": ["Czekolada", "Chleb"]
```

Comment:
I know this is just a stub, but c`mon. All other properties are set where they should be.